### PR TITLE
getting boot node ip fault tolerant

### PR DESCRIPTION
**Pull request by x-wf**

### CODE REVIEW

**Comment by x-wf** [comment]
#30

File: tools/libcloud/gcp.py [hunk 1]
```diff
@@ -175,7 +175,8 @@ def get_boot_node(gce):
     nodes = gce.list_nodes()
     for node in nodes:
         if node.name.startswith("core-boot-"):
-            gce.wait_until_running([node], 5, 600) # 5 second loops | 10 minute timeout
+            while(not node.public_ips):
```
Comment (blenessy) [inline tools/libcloud/gcp.py:178]: nope, you need to refresh the `node` metadata or you'll be stuck in this while loop forever:

```python
while not node.public_ips:
    logging.info("Waiting for %s to get its public IP ...", node.name)
    time.sleep(5)
    node = gce.ex_get_node(node.name)
 ```

**Review by blenessy — CHANGES_REQUESTED** [review]

**Review by swiec — CHANGES_REQUESTED** [review]
Oops, looks like you're fixing wrong line, take a look at logs attached to issue
` File "libcloud/mtps.py", line 166, in <module>
    os.environ["RADIX_MTPS_NETWORK_SEEDS"] = boot_node.public_ips[0]`
problem is in the line 166

**Comment by blenessy** [comment]
superseded by #68

**Pull request closed without merging**